mdsuite/file_io/trajectory_files.py

```diff
@@ -125,7 +122,8 @@ def build_file_structure(self, batch_size: int = None):
             if self.sort:
                 positions = np.array(self.experiment.species[item]['indices'])
             else:
-                positions = np.array([np.array(self.experiment.species[item]['indices']) + i * self.experiment.number_of_atoms -
+                positions = np.array([np.array(self.experiment.species[item]['indices']) + i *
+                                      self.experiment.number_of_atoms -
```

Comment:
List comprehension over 3 lines is also hard to read, but not formatting issue here.